update.php

```diff
@@ -747,4 +747,14 @@ function pre_update_1365()
 		return Update::FAILED;
 	}
 
+	return Update::SUCCESS;
+}
+
+function update_1374()
+{
+	if (!DBA::e("UPDATE `item` SET `thr-parent` = `parent-uri` WHERE `thr-parent` = ''")) {
```

Comment:
Please also set the `thr-parent-id` with the `parent-uri-id` for these cases.

Reply:
Done.